### ISSUE TITLE
Training/Validation Split in Data

### ISSUE THREAD

**Issue by Chatha-Sphere** [Chatha-Sphere/pno-ai#5]
How will this trade be affected by the various data augmentation strategies?

**Comment by Chatha-Sphere** [comment]
From what I understand, the idea is to compare training loss vs. validation loss every "x" number of iterations (say 1,000).

Following magenta's example, start with a 90/10 training/validation split. Only augment (transpositions, tempo stretches) the training data.
